src/launch/types.rs

```diff
@@ -1,5 +1,36 @@
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::certs::sev;
+use crate::launch::{Policy, Session};
+
+use std::marker::PhantomData;
+
 /// Initialize the SEV platform context.
 #[repr(C)]
 pub struct Init;
+
+/// Initiate SEV launch flow.
+#[repr(C)]
+pub struct LaunchStart<'a> {
+    pub handle: u32,
```

Comment:
Rather than making `handle` `pub`, we can just `impl From<LaunchStart> for Handle`. It is only an output parameter.